PLUGINS.md

```diff
@@ -940,19 +940,52 @@ def cmdr_data(data, is_beta):
 | :-------- | :--------------: | :------------------------------------------------------------------------------------------------------- |
 | `data`    |     `CAPIData`   | `/profile` API response, with `/market` and `/shipyard` added under the keys `marketdata` and `shipdata` |
 | `is_beta` |      `bool`      | If the game is currently in beta                                                                         |
+
+If a plugin has a `capi_fleetcarrier()` function it gets called when the application has just fetched fresh Fleetcarrier data from Frontier's CAPI servers. This is done when `CarrierBuy`, `CarrierStats` or `CarrierTradeOrder` events are detected in the Player Journal. To avoid flooding Frontier's CAPI server, a throttle is applied to ensure a significant interval between requests (currently 15 mins).
+
```

Comment:
Add a note here about how a single query can take a long time under some circumstances.  This means not only could a fleetcarrier update take a long time, *but it will also possibly delay all other CAPI queries*.

Reply:
Also, now I realise we might have lost some "don't allow 'Update' if already in progress" check when I made CAPI threaded.  But don't worry about that for this PR.

Reply:
Will add some additional text.